src/lib/Components/Buttons/CaretButton.tsx

```diff
@@ -10,15 +10,16 @@ interface Props {
 export const CaretButton: React.SFC<Props> = ({ text, onPress }) => {
   return (
     <TouchableWithoutFeedback onPress={onPress}>
-      <Flex alignItems="center" justifyContent="space-between" flexDirection="row" height={36}>
+      <Flex flexDirection="row" align-items="base-line">
         <Sans size="3t" weight="medium">
           {text}
-          {"  "}
+        </Sans>
+        <Box ml={1}>
           <Image
-            style={{ alignSelf: "center", tintColor: color("black30") }}
+            style={{ tintColor: color("black30"), top: 3 }}
```

Comment:
rather than `top: 3` can add some padding (`pb`) to Box and use our [theme](https://github.com/artsy/palette/blob/master/src/Theme.tsx#L119) definition -- `pb={.3}`

Reply:
I dont think the RN image element can take a `pb` prop but I'll try it out...

Reply:
Can put it on `<Box>`, since right now the image is pushing up, can use the box to push down (might work?)

Reply:
Doesn't work, going to leave this one alone.